code/LINDA/LINDA_turf_tile.dm

```diff
@@ -6,6 +6,7 @@
 	var/atmos_adjacent_turfs = 0
 	var/atmos_adjacent_turfs_amount = 0
 	var/atmos_supeconductivity = 0
+	var/atoms_overlay_type = ""

```

Comment:
don't recreate this variable, use the one in gas mixtures datum.
Also, if its better, remove the one from gas mixtures and let this one exist, no other atom uses it if I'm not wrong, but doublecheck that because I'm not sure.
About performance, I don't remember how the gas mixtures edit the var, so thats why im saying.